rtscomponentkit/src/main/java/io/dolby/rtscomponentkit/data/RTSViewerDataStore.kt

```diff
@@ -45,14 +47,14 @@ class RTSViewerDataStore(
             _statistics.value = null
         }
 
-        override fun onTrack(track: VideoTrack, p1: Optional<String>) {
+        override fun onTrack(track: VideoTrack, p1: Optional<String>?) {
```

Comment:
Having an Optional<>? makes no sense (btw the sdk tells that p1, the mid, can be null, it actually can't be null - but for this one, I'll need to check a bit more when we are the sender)